packages/surveys/src/utils/results-availability/migrate-form-properties-settings.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  getProp,
+  cloneObject,
+  IGetSurveyModelsResponse
+} from "@esri/hub-common";
+
+/**
+ * Populate form item properties is missing
```

Comment:
Is it valuable to provide more context for why this might occur? When would a dev use this method (or is it effectively 'internal').